From 29845be64dc1470060388c2fb32223ff42af8d25 Mon Sep 17 00:00:00 2001
From: Daniel T Chen <crimsun@ubuntu.com>
Date: Wed, 10 Mar 2010 18:14:11 -0500
Subject: [PATCH 163/163] Handle 'Digital Mic' as an 'Input Source'

BugLink: https://launchpad.net/bugs/533877

Some laptops have 'Digital Mic' exposed as an 'Input Source', e.g., Dell
XPS 1330, so handle these, too.
---
 .../alsa/mixer/paths/analog-input.conf.common      |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/src/modules/alsa/mixer/paths/analog-input.conf.common b/src/modules/alsa/mixer/paths/analog-input.conf.common
index 66c3dda..6c5fae9 100644
--- a/src/modules/alsa/mixer/paths/analog-input.conf.common
+++ b/src/modules/alsa/mixer/paths/analog-input.conf.common
@@ -62,6 +62,10 @@ priority = 5
 [Element Input Source]
 enumeration = select
 
+[Option Input Source:Digital Mic]
+name = input-microphone
+priority = 20
+
 [Option Input Source:Mic]
 name = input-microphone
 priority = 20
-- 
1.7.0.2

